src/gluonts/torch/model/deepar/estimator.py

```diff
@@ -126,6 +126,10 @@ class DeepAREstimator(PyTorchLightningEstimator):
         (default: ``NegativeLogLikelihood()``).
     scaling
         Whether to automatically scale the target values (default: true).
+    default_scale
+        Default scale that is applied if the context length window is
+        completely unobserved. If not set, the scale in this case will be
+        the mean scale in the batch.
```

Comment:
I think we probably want to fix this: having the model behaviour on one batch element depend on what the rest of the batch looks like is probably not desirable, right?

Reply:
I think the reason for using batch statistics for default_scale makes sense from the point of the computation of the average batch loss. A single time series on a very high scale, if not scaled with appropriate a batch statistic, can mess up the loss, no?